sources/engine/Stride.BepuPhysics/Stride.BepuPhysics/BodyComponent.cs

```diff
@@ -167,6 +192,14 @@ public Vector3 LinearVelocity
         }
     }
 
+    /// <summary>
+    /// The rotation velocity in unit per second, also known as torque
```

Comment:
Torque refers to an angular force, not an angular velocity, isn't it?

Reply:
Yep, the distinction isn't huge in this context but best avoid any confusion, removed it from the summary